### PR TITLE
Update MemberProfileExtension.php

### PR DESCRIPTION
**Pull request by AntonyThorpe**
FIX for the unserialize function when passing null to the first parameter.  Thanks.

### CODE REVIEW

File: src/Extensions/MemberProfileExtension.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ class MemberProfileExtension extends DataExtension
 
     public function getPublicFields()
     {
-        return (array) unserialize($this->owner->getField('PublicFieldsRaw'));
+        return (array) unserialize($this->owner->getField('PublicFieldsRaw') '');
```
Comment (michalkleiner) [inline src/Extensions/MemberProfileExtension.php:36]:
```suggestion
        return (array) unserialize($this->owner->getField('PublicFieldsRaw') ?? '');
```

**Review by michalkleiner — CHANGES_REQUESTED** [review]

File: src/Extensions/MemberProfileExtension.php (hunk 1 above)
Comment (AntonyThorpe) [reply src/Extensions/MemberProfileExtension.php]: Thank you.  Have added the missing ?? to the PR.

**Comment by AntonyThorpe** [comment]
Any feedback please?

**Comment by nyeholt** [comment]
Handled in #190

**Pull request closed without merging**